website/pkgdown.yml

```diff
@@ -1,65 +1,65 @@
-pandoc: 2.9.2.1
-pkgdown: 1.5.1
+pandoc: 2.11.4
+pkgdown: 1.6.1
 pkgdown_sha: ~
 articles:
   applications: applications.html
   custom_layers: custom_layers.html
   custom_models: custom_models.html
-  addition_rnn: examples/addition_rnn.html
-  babi_memnn: examples/babi_memnn.html
-  babi_rnn: examples/babi_rnn.html
-  cifar10_cnn: examples/cifar10_cnn.html
-  cifar10_densenet: examples/cifar10_densenet.html
-  cifar10_resnet: examples/cifar10_resnet.html
-  conv_filter_visualization: examples/conv_filter_visualization.html
-  conv_lstm: examples/conv_lstm.html
-  deep_dream: examples/deep_dream.html
-  eager_cvae: examples/eager_cvae.html
-  eager_dcgan: examples/eager_dcgan.html
-  eager_image_captioning: examples/eager_image_captioning.html
-  eager_pix2pix: examples/eager_pix2pix.html
-  eager_styletransfer: examples/eager_styletransfer.html
-  fine_tuning: examples/fine_tuning.html
-  image_ocr: examples/image_ocr.html
-  imdb_bidirectional_lstm: examples/imdb_bidirectional_lstm.html
-  imdb_cnn: examples/imdb_cnn.html
-  imdb_cnn_lstm: examples/imdb_cnn_lstm.html
-  imdb_fasttext: examples/imdb_fasttext.html
-  imdb_lstm: examples/imdb_lstm.html
-  index: examples/index.html
-  lstm_benchmark: examples/lstm_benchmark.html
-  lstm_seq2seq: examples/lstm_seq2seq.html
-  lstm_stateful: examples/lstm_stateful.html
-  lstm_text_generation: examples/lstm_text_generation.html
-  mmd_cvae: examples/mmd_cvae.html
-  mnist_acgan: examples/mnist_acgan.html
-  mnist_antirectifier: examples/mnist_antirectifier.html
-  mnist_cnn: examples/mnist_cnn.html
-  mnist_cnn_embeddings: examples/mnist_cnn_embeddings.html
-  mnist_dataset_api: examples/mnist_dataset_api.html
-  mnist_hierarchical_rnn: examples/mnist_hierarchical_rnn.html
-  mnist_irnn: examples/mnist_irnn.html
-  mnist_mlp: examples/mnist_mlp.html
-  mnist_net2net: examples/mnist_net2net.html
-  mnist_siamese_graph: examples/mnist_siamese_graph.html
-  mnist_swwae: examples/mnist_swwae.html
-  mnist_tfrecord: examples/mnist_tfrecord.html
-  mnist_transfer_cnn: examples/mnist_transfer_cnn.html
-  neural_style_transfer: examples/neural_style_transfer.html
-  nmt_attention: examples/nmt_attention.html
-  nueral_doodle: examples/nueral_doodle.html
-  pretrained_word_embeddings: examples/pretrained_word_embeddings.html
-  quora_siamese_lstm: examples/quora_siamese_lstm.html
-  reuters_mlp: examples/reuters_mlp.html
-  reuters_mlp_relu_vs_selu: examples/reuters_mlp_relu_vs_selu.html
-  stateful_lstm: examples/stateful_lstm.html
-  text_explanation_lime: examples/text_explanation_lime.html
-  tfprob_vae: examples/tfprob_vae.html
-  unet: examples/unet.html
-  unet_linux: examples/unet_linux.html
-  variational_autoencoder: examples/variational_autoencoder.html
-  variational_autoencoder_deconv: examples/variational_autoencoder_deconv.html
-  vq_vae: examples/vq_vae.html
+  examples/addition_rnn: addition_rnn.html
```

Comment:
Maybe investigate why this is really needed? Is it a pkgdown change?

Reply:
I'm not sure it's needed. It's a change that `pkgdown::build_site()` made. Maybe I should to rollback to a previous version of pkgdown?

Reply:
I think that's fine them. All pages seems to still exist and the their path didn't change either.